bosh/releases/generators/pre_render_scripts/generator.sh

```diff
@@ -4,7 +4,9 @@ set -o errexit -o nounset
 
 touch "${OUTPUT}"
 
-echo "- type: replace" >> "${OUTPUT}"
-echo "  path: /instance_groups/name=${INSTANCE_GROUP}/jobs/name=${JOB}/properties/bosh_containerization?/pre_render_scripts/-" >> "${OUTPUT}"
-echo "  value: |" >> "${OUTPUT}"
-cat "${PRE_RENDER_SCRIPT}" | sed 's/^/    /' >> "${OUTPUT}"
+{
+  echo "- type: replace";
+  echo "  path: /instance_groups/name=${INSTANCE_GROUP}/jobs/name=${JOB}/properties/bosh_containerization?/pre_render_scripts/-";
+  echo "  value: |";
+  sed 's/^/    /' < "${PRE_RENDER_SCRIPT}";
+} >> "${OUTPUT}"
```

Comment:
Using heredoc might be nice (but totally optional); untested:

```sh
{
cat <<EOF
- type: replace
  path: /instance_groups/name=${INSTANCE_GROUP}/jobs/name=${JOB}/properties/bosh_containerization?/pre_render_scripts/-
  value: |
EOF
sed 's/^/    /' < "${PRE_RENDER_SCRIPT}"
} >> "${OUTPUT}
```

(I don't believe you need the explicit semicolons, even in a `{}` brace, if you have new lines.)

Reply:
Yeah, looks better. And I just realized I don't need the `touch` above, I can simply `... > "${OUTPUT}"`.